From 1f5bc7c02e7d5105da67ce2a2cac43c04ebb0792 Mon Sep 17 00:00:00 2001
From: Ionut Nechita <ionut_n2001@yahoo.com>
Date: Tue, 8 Mar 2022 08:31:41 +0200
Subject: [PATCH 11/58] SUNLIGHT: Minor fix in Makefile for headers

Description:
 - Minor fix in Makefile for headers
 - Include folder: sunlight/include

Bug: N/A
Change-Id: I39ac2a6525443ad07980f3aaf7c995f0ac3c30f0
Signed-off-by: Ionut Nechita <ionut_n2001@yahoo.com>
---
 Makefile | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index bbb4ce1a0de7..8ace6dbad9ac 100644
--- a/Makefile
+++ b/Makefile
@@ -509,13 +509,6 @@ CFLAGS_KERNEL	=
 AFLAGS_KERNEL	=
 LDFLAGS_vmlinux =
 
-# Prefer linux-backports-modules
-ifneq ($(KBUILD_SRC),)
-ifneq ($(shell if test -e $(KBUILD_OUTPUT)/sunlight-build; then echo yes; fi),yes)
-SUNLIGHTINCLUDE := -I/usr/src/linux-headers-lbm-$(KERNELRELEASE)
-endif
-endif
-
 # Use USERINCLUDE when you must reference the UAPI directories only.
 USERINCLUDE    := \
 		-I$(srctree)/arch/$(SRCARCH)/include/uapi \
@@ -528,7 +521,6 @@ USERINCLUDE    := \
 # Use LINUXINCLUDE when you must reference the include/ directory.
 # Needed to be compatible with the O= option
 LINUXINCLUDE    := \
-		$(SUNLIGHTINCLUDE) \
 		-I$(srctree)/arch/$(SRCARCH)/include \
 		-I$(objtree)/arch/$(SRCARCH)/include/generated \
 		$(if $(building_out_of_srctree),-I$(srctree)/include) \
@@ -536,7 +528,7 @@ LINUXINCLUDE    := \
 		$(USERINCLUDE)
 
 # SUNLIGHT: Include our third party driver stuff too
-LINUXINCLUDE   += -Isunlight/include $(if $(KBUILD_SRC),-I$(srctree)/sunlight/include)
+LINUXINCLUDE   += -I$(srctree)/sunlight/include
 
 KBUILD_AFLAGS   := -D__ASSEMBLY__ -fno-PIE
 KBUILD_CFLAGS   := -Wall -Wundef -Werror=strict-prototypes -Wno-trigraphs \
@@ -1187,7 +1179,7 @@ ifeq ($(KBUILD_EXTMOD),)
 endif
 	$(Q)$(MAKE) $(hdr-inst)=$(hdr-prefix)include/uapi
 	$(Q)$(MAKE) $(hdr-inst)=$(hdr-prefix)arch/$(SRCARCH)/include/uapi
-	$(Q)$(MAKE) $(hdr-inst)=sunlight/include dst=include oldheaders=
+	$(Q)$(MAKE) $(hdr-inst)=$(hdr-prefix)sunlight/include dst=include oldheaders=
 
 ifeq ($(KBUILD_EXTMOD),)
 core-y		+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/ block/
-- 
2.35.1

